spec/migrations/20170710132935_custom_button_to_classes_spec.rb

```diff
@@ -0,0 +1,30 @@
+require_migration
+
+describe CustomButtonToClasses do
+  let(:custom_button_stub) { migration_stub(:CustomButton) }
+
+  migration_context :up do
+    it "sets the image class and color" do
+      custom_button = custom_button_stub.create!(:options => {:button_image => 1})
+      migrate
+
+      custom_button.reload
+
+      expect(custom_button.options).to have_key(:button_image_class)
+      expect(custom_button.options).not_to have_key(:button_image)
```

Comment:
how about add a case (another button) where you will have both `:button_image_class` and `:button_image_color`?